src/nft/components/collection/CollectionStats.tsx

```diff
@@ -251,18 +251,18 @@ const StatsItem = ({ children, label, isMobile }: { children: ReactNode; label:
 
 const StatsRow = ({ stats, isMobile, ...props }: { stats: GenieCollection; isMobile?: boolean } & BoxProps) => {
   const uniqueOwnersPercentage = stats.stats
-    ? roundWholePercentage((stats.stats.num_owners / stats.stats.total_supply) * 100)
+    ? roundWholePercentage(((stats.stats.num_owners ?? 1) / (stats.stats.total_supply ?? 0)) * 100)
     : 0
-  const totalSupplyStr = stats.stats ? quantityFormatter(stats.stats.total_supply) : 0
+  const totalSupplyStr = stats.stats ? quantityFormatter(stats.stats.total_supply ?? 0) : 0
   const listedPercentageStr =
-    stats.stats && stats.stats.total_listings > 0
-      ? roundWholePercentage((stats.stats.total_listings / stats.stats.total_supply) * 100)
+    stats.stats && stats.stats.total_listings
+      ? roundWholePercentage((stats.stats.total_listings / (stats.stats.total_supply ?? 0)) * 100)
```

Comment:
division by 0 fallback is spooki

Reply:
mixed up the divisor and dividend. fixed.